app/assets/stylesheets/companion_window.css

```diff
@@ -101,6 +101,11 @@ body {
     display: none !important;
   }
 
+  .locked-status img {
```

Comment:
Shouldn't this go in the `model.css`?

Reply:
I considered it but I think it makes sense to have it be consistent across viewers. (PDF also uses it)

Reply:
How was the image for the PDF formatted prior to this change?

Reply:
Ah. It was in it's own file. I will go ahead an move that into the 3d 
```@import url("locked_status.css");```